h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -320,7 +320,11 @@ void planWork() {
 
     for (Algo algo : Algo.values()) {
       if (!skippedAlgos.contains(algo) && !algo.enabled()) {
-        _eventLog.warn(Stage.ModelTraining, "AutoML: "+algo.name()+" is not available; skipping it.");
+        boolean isMultinode = H2O.CLOUD.size() > 1;
+        _eventLog.warn(Stage.ModelTraining,
+                isMultinode ? "AutoML: "+algo.name()+" is not available in multi-node cluster; skipping it." :
```

Comment:
Maybe we can add info how to enable it manually? In such case we either link documentation or create a technote - eg. https://0xdata.atlassian.net/browse/TN-1

Reply:
yes, was wondering how to expose this... the link to doc should be fine. Adding it.

Reply:
Are technotes public? Good to know